pkgs/applications/misc/redshift/default.nix

```diff
@@ -7,6 +7,7 @@
 , withRandr ? stdenv.isLinux, libxcb
 , withDrm ? stdenv.isLinux, libdrm
 
+, nixosTests
```

Comment:
I guess you meant to add this to passthru?

Reply:
I will remove it. I tried to add this to `passthru.tests` but since redshift is have 3 different derivations and my tests doesn't support it, I decided for now to not add the tests to `passthru.tests`.